### ISSUE TITLE
Gradio gets stuck in loop after clicking an image or button

### ISSUE THREAD

**Issue by Jallisto** [Zeqiang-Lai/DragGAN#115]
Gradio gets stuck in an infinite loop after clicking on an image or when clicking a button on the panel. The image disappears immediately when doing so.

CPU and GPU activity sticks at only ~5% (10% tops) with Gradio running. So although it's continually loading not much effort seems to be put into it.

Does anyone know what's wrong or how to solve this?

The setup I'm currently using:

Windows 10
Visual Studio 2019 (Community)
Miniconda3
NVIDIA GeForce GTX 1080
CUDA Toolkit v11.7 (installed separately from Miniconda3)

Packages (incomprehensive list)
draggan 1.1.6
gradio 3.34.0
gradio-client 0.2.6
python 3.7.16
torch 1.13.1+cu117
torchaudio 0.13.1+cu117
torchvision 0.1.6

![Gradio infinite loop 2023_08_27](https://github.com/OpenGVLab/DragGAN/assets/143288377/8a424957-2fe0-4658-9d3d-37d084755b09)

**Comment by Zeqiang-Lai** [comment]
The latest code works on my computer. Your issue might be caused by the network connection issues of huggingface gradio.

**Comment by Jallisto** [comment]
> The latest code works on my computer. Your issue might be caused by the network connection issues of huggingface gradio.

Thanks for the suggestion!
I'll see if someone at Hugginface / Gradio knows what's wrong.

**Comment by mindy0-0** [comment]
I was wondering if you have solved the problem now?

**Comment by Jallisto** [comment]
> I was wondering if you have solved the problem now?

No, I haven't had a response back from the Huggingface forum and I personally don't know how to fix it.
If I know a solution, I'll post it here.

**Comment by mindy0-0** [comment]
> > I was wondering if you have solved the problem now?
> 
> No, I haven't had a response back from the Huggingface forum and I personally don't know how to fix it. If I know a solution, I'll post it here.

For you can have a try in the installation environment/site_packages gradio/themes/utils/fonts. Py commented in this web site, https://fonts.googleapis.com/, and then remove the browser's cache to try. I tried it and found that it could run.

**Comment by MADM1k33** [comment]
Still not fixed
